python/ray/train/__init__.py

```diff
@@ -1,3 +1,15 @@
+# Try import ray[train] core requirements (defined in setup.py)
+try:
+    import pandas  # noqa: F401
+    import requests  # noqa: F401
+    import pyarrow  # noqa: F401
+except ImportError as exc:
+    raise ImportError(
+        "Can't import ray.train as some dependencies are missing. "
+        "Run `pip install ray[train]` to fix."
```

Comment:
can we make the command is `pip install "ray[train]"`

I think `ray[train]` by itself doesn't work on some shells.

Reply:
Same for below